docroot/themes/humsci/humsci_basic/src/scss/components/_main-nav.scss

```diff
@@ -1,105 +1,354 @@
+        font-size: hb-calculate-rems(30px);
       }
 
-      @include grid-media('lg') {
-        .su-main-nav__menu-container {
-          position: absolute;
+      @include grid-media-min('lg') {
+        .js & {
+          font-size: hb-calculate-rems(18px);
+          font-weight: hb-theme-font-weight(semibold);
+          display: flex;
+          align-items: flex-end;
+          border: 0;
+
+          &[aria-hidden="false"] {
+            margin: 0;
+          }
         }
       }
     }
 
     &-lv2 {
-      @include grid-media('lg') {
-        // Hide any nested menus that are level 3 or greater
-        .su-main-nav__button,
-        .su-main-nav__menu-container {
-          display: none;
+      font-weight: hb-theme-font-weight(regular);
+      font-size: hb-calculate-rems(18px);
+
+      @include grid-media-min('md') {
+        font-size: hb-calculate-rems(22px);
+      }
+
+      @include grid-media-min('lg') {
+        .js & {
+          font-weight: hb-theme-font-weight(semibold);
+          font-size: inherit;
         }
       }
     }
-  }
 
-  &__menu-container {
-    padding-left: hb-calculate-rems(20px);
-    margin-top: hb-calculate-rems(8px);
-    margin-bottom: hb-calculate-rems(8px);
-
-    &[aria-hidden="true"] {
-      display: none;
+    &-lv3 {
+      @include grid-media-min('lg') {
+        .js & {
+          display: none;
+        }
+      }
     }
   }
 
   &__item {
     position: relative;
+
+    .hb-main-nav__menu-lv1 > & {
+      border-top: $hb-menu-light-border;
+
+      @include grid-media-min('lg') {
+        .js & {
+          margin-right: hb-calculate-rems(24px);
+          border: 0;
+        }
+      }
+    }
+
+    .hb-main-nav__menu-lv2 &:not(:last-child) {
+      border-bottom: $hb-menu-light-border;
+
+      @include grid-media-min('lg') {
+        .js & {
+          border: 0;
+        }
+      }
+    }
   }
 
   &__link {
-    &::after {
-      display: none !important;
+    position: relative;
+    display: block;
+    padding: hb-calculate-rems(21px) hb-calculate-rems(17px);
+
+    color: $hb-color--black;
+    text-decoration: none;
+    line-height: 120.3%;
+
+    @include grid-media-min('md') {
+      line-height: 128%;
+    }
+
+    @include grid-media-min('lg') {
+      .js & {
+        padding: hb-calculate-rems(10px) 0 hb-calculate-rems(20px);
+        line-height: 110%;
+      }
+    }
+
+    .hb-main-nav__item--parent > & {
+      padding: hb-calculate-rems(21px) hb-calculate-rems(48px) hb-calculate-rems(21px) hb-calculate-rems(17px);
+
+      @include grid-media-min('lg') {
+        .js & {
+          padding: hb-calculate-rems(10px) hb-calculate-rems(20px) hb-calculate-rems(20px) 0;
+        }
+      }
+    }
+
+    &:hover:not(.is-active) {
+      background-color: $hb-menu-link-background-hover;
+    }
+
+    // Targeting only the first level of navigation on desktop
+    .hb-main-nav__menu-lv1 > li > & {
+      @include grid-media-min('lg') {
+        .js &::before {
```

Comment:
can you indent the &:before to the next line?

Reply:
That's what I originally had but the linter yelled at me for too many levels of nested code.

Reply:
ugh